source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/readonly/TextRendererUtil.java

```diff
@@ -130,68 +131,88 @@ else if (textAlignment == HorizontalAlignment.Right)
         return alignment;
     }
 
-    static TextSize getTextSizeFromStyle(HostConfig hostConfig, TextStyle textStyle, TextSize override)
+    static TextSize computeTextSize(HostConfig hostConfig, RenderArgs renderArgs, TextStyle declaredStyle, TextSize declaredSize)
```

Comment:
Actually, `getTextSizeFromStyle` was part of the heading styles feature, so it still hasn't gone out in a public release. I think we should avoid using `get` and `set` method names unless they are getting/setting properties on this object. Kotlin treats get/set prefixed methods in a special way, so it may be confusing for hosts consuming the API.